tests/unit/services/flash-messages-test.js

```diff
@@ -162,6 +162,29 @@ test('it sets the correct defaults for service properties', function(assert) {
   }
 });
 
+test('when preventDuplicates is `false` setting a message is not required', function(assert) {
+  set(service, 'defaultPreventDuplicates', false);
+
+  service.add({
+    customProperty: 'ohai'
+  });
+
+  assert.equal(get(service, 'queue.0.customProperty'), 'ohai');
```

Comment:
Minor nitpick, but rather than `queue.0` could we do `queue.firstObject`? I feel it reads just a bit more clear

Reply:
yup, done :)